lib/types/enums.dart

```diff
@@ -6,4 +8,34 @@
 enum AlertBannerLocation {
   top,
   bottom,
+  endButtom,
+  endTop,
+  startButtom,
+  startTop;
```

Comment:
Should these names be changed?

For example, `endButton` locates the banner at the end horizontally at the bottom of the screen, thus, I believe it should be named `endBottom`. This will make it more intuative.

The same principle applies for `endButton`, `endTop`, `startButton`, and `startTop`.